scripts/apps/archive/controllers/BaseListController.js

```diff
@@ -8,9 +8,10 @@ export class BaseListController {
         $scope.selected = {};
 
         $scope.fetchNext = (from) => {
-            var criteria = this.getQuery();
+            var criteria = this.getQuery(null, $scope.repo.archive || false);
```

Comment:
I would call this `source` instead of `criteria`. It is confusing when you look at it from the perspective of the `fetchItems` function where `criteria` means the parent object.